modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -404,7 +404,7 @@ a|`--save-partindex <id>...`
 a|Save partitions with this number or range.
 
 a|`--insecure`
-a|Skip signature verification.
+a|Skip RHCOS image signature verification.
```

Comment:
This should be `{op-system} image` shouldn't it?

Reply:
Thanks, @zaneb . Correct. I updated the text based on your suggestion.